src/lib/Navbar.svelte

```diff
@@ -40,6 +40,11 @@
 		text-decoration: inherit;
 		margin-left: .8rem;
 		margin-right: .8rem;
+		transition: background-color 0.2s ease-in-out;
+	}
+
+	.links a:hover {
+		color: #666666;
```

Comment:
Is there any reason why we use a brightness filter in the previous element and not here?